src/components/Type2020/Blueprint.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Blueprint for Q2 2020 Type transition to new font sizes and styles.
+ *
+ *
+ * Name proposals
+ * --------------------------
+ * Type refers to the current soon to be "legacy" Type components.
+ * We need something to differentiate from our current Type component while transitioning.
+ *   - Type2020 (working prototype in this doc & files)
+ *   - Type2020 / TwentyType
+ *   - TypeNew / NewType
```

Comment:
I like `TypeNew` or better `Type2` better then the 2020 version

Reply:
`Type2` seems easy.